public/components/content-list-drawer/content-list-drawer.js

```diff
@@ -28,6 +28,13 @@ export function wfContentListDrawer($rootScope, config, $timeout, $window, conte
             });
     }
 
+    function buildIndesignUrl(fields) {
+        return config.indesignExportUrl
```

Comment:
You could avoid duplicating the logic in the indesign/incopy functions with a something like 
```
   function buildExportUrl(baseExportUrl) {
        return baseExportUrl
            .replace(/\$\{(.*?)\}/g, function(match, fieldName) {
                return fields[fieldName] || "";
            });
}
```